_data/team/roles/continuous_integration.yml

```
@@ -0,0 +1,13 @@
+role: Continuous integration
+tasks:
+  - Building and developing new CI infrastructure
+  - Monitoring CI status
+  - Maintenance and fixes
+current_leads:
+  - Irfan Alibay
+  - Richard Gowers
+current_members:
+  - Fiona Naughton
+historical_members:
+  - Irfan Alibay
```

```suggestion
  - Irfan Alibay (2020-2023)
```
I also note that @tylerjereddy should be here too.

---

I think the main historical things for me are: CI, portability to Windows platform, and Python 2.7->3.6 transition. IIRC I was writing some `asv` benchmarks early on as well. That said, I'm not particularly concerned about being listed in X places, I still say I'm a core developer and if someone on the team pings me for help on something I'd generally intend to provide assistance if I can, even though I'm admittedly putting more attention upstream at the moment.

---

Thanks Tyler! After some discussion we realised we still have some ways to go in working out how best to acknowledge historical contributors, so in our initial version of this PR we will just stick with publishing the current team. I'll resolve this conversation for now and we can re-visit when we come back to historical contributors in the future.